public/data/challenges/ERC20.json

```diff
@@ -10,7 +10,7 @@
ber: 1\n      }, this), \"\\n\", _jsxDEV(_components.li, {\n        children: [\"\\n\", _jsxDEV(_components.p, {\n          children: [_jsxDEV(_components.strong, {\n            children: \"TransferFrom Function:\"\n          }, undefined, false, {\n            fileName: \"<source.js>\",\n            lineNumber: 79,\n            columnNumber: 4\n          }, this), \" Implement the \", _jsxDEV(_components.code, {\n            children: \"transferFrom\"\n          }, undefined, false, {\n            fileName: \"<source.js>\",\n            lineNumber: 79,\n            columnNumber: 45\n          }, this), \" function to allow a spender to transfer tokens from one address to another, using an allowance set by the \", _jsxDEV(_components.code, {\n            children: \"approve\"\n          }, undefined, false, {\n            fileName: \"<source.js>\",\n            lineNumber: 79,\n            columnNumber: 166\n          }, this), \" function. Override the \", _jsxDEV(_components.code, {\n            children: \"transferFrom\"\n          }, undefined, false, {\n            fileName: \"<source.js>\",\n            lineNumber: 79,\n            columnNumber: 199\n          }, this), \" function and use the \", _jsxDEV(_components.code, {\n            children: \"_transfer\"\n          }, undefined, false, {\n            fileName: \"<source.js>\",\n            lineNumber: 79,\n            columnNumber: 235\n          }, this), \" and \", _jsxDEV(_components.code, {\n            children: \"_approve\"\n          }, undefined, false, {\n            fileName: \"<source.js>\",\n            lineNumber: 79,\n            columnNumber: 251\n          }, this), \" functions to implement this functionality.\"]\n        }, undefined, true, {\n          fileName: \"<source.js>\",\n          lineNumber: 79,\n          columnNumber: 4\n        }, this), \"\\n\"]\n      }, undefined, true, {\n        fileName: \"<source.js>\",\n        lineNumber: 79,\n        columnNumber: 1\n      }, this), \"\\n\"]\n    }, undefined, true, {\n      fileName: \"<source.js>\",\n      lineNumber: 69,\n      columnNumber: 1\n    }, this)]\n  }, undefined, true, {\n    fileName: \"<source.js>\",\n    lineNumber: 1,\n    columnNumber: 1\n  }, this);\n}\nfunction MDXContent(props = {}) {\n  const {wrapper: MDXLayout} = Object.assign({}, _provideComponents(), props.components);\n  return MDXLayout ? _jsxDEV(MDXLayout, Object.assign({}, props, {\n    children: _jsxDEV(_createMdxContent, props, undefined, false, {\n      fileName: \"<source.js>\"\n    }, this)\n  }), undefined, false, {\n    fileName: \"<source.js>\"\n  }, this) : _createMdxContent(props);\n}\nreturn {\n  default: MDXContent\n};\nfunction _missingMdxReference(id, component, place) {\n  throw new Error(\"Expected \" + (component ? \"component\" : \"object\") + \" `\" + id + \"` to be defined: you likely forgot to import, pass, or provide it.\" + (place ? \"\\nIt’s referenced in your code at `\" + place + \"`\" : \"\"));\n}\n",
```

Comment:
Seems like dev environment fragments generated. Will need to run script prior to pushing